### PR TITLE
AAP-9897 updated section 2.1

### PR DESCRIPTION
**Pull request by emurtoug**
[AAP-9897](https://issues.redhat.com/browse/AAP-9897)

https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/getting_started_with_automation_hub/proc-configure-automation-hub-server 

Edited master.adoc and added section 2.1 Configuring Red Hat automation hub as the primary source for content using the CLI

### CODE REVIEW

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+====
+
+.Procedure
+
+. Navigate to your Automation Controller. 
+. Create a new credential.
+.. Click btn:[Add] from the **Credentials** screen.
+.. Enter the name for your new credential in the **Name** field.
+.. Optional: enter a description and enter or select the name of the organization with which the credential is associated.
+.. Under Organization, select the organization that you would like to use the credential for.
```
Comment (sayjadha) [inline downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc:22]: Should 'Organization' and 'Galaxy Credentials' be in bold formatting, like the rest of the UI elements that are starting with 'Under ...' ? Please check the UI elements guidelines in the IBM Style Guide (pg. 215) and the screen and make the necessary formatting changes.

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+.. Click btn:[Edit].
+.. Under Galaxy Credentials, click image:search-icon.png.
+.. Select the credential you created for automation hub, and place it at the beginning of the list.
+.. Optional: If you have a secondary source of content, such as Ansible Galaxy, place this credential after the credential you created for automation hub. 
+.. Click btn:[Select].
+.. Click btn:[Save].
+
+.Verification
+
+To validate the credential, update an existing source control management (SCM)-based project by selecting the project and clicking the [refresh icon] button.
```
Comment (sayjadha) [inline downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc:38]: Is this `[refresh icon] button` -> **Refresh** icon?
Same comment for 'Update the project by clicking the refresh icon [refresh icon].' text.

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+. Navigate to your project repository.
+. Select a project that uses a collections/requirements.yml file.
```
Comment (sayjadha) [inline downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc:41]: File names should be highlighted in monospace font. Please see the 'Files and directories' section in the IBM Style Guide (pg. 206).

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+.. Optional: enter a description and enter or select the name of the organization with which the credential is associated.
+.. Under Organization, select the organization that you would like to use the credential for.
+.. Select **Ansible Galaxy/Automation Hub API Token** as the credential type.
+.. Under **Type Details**, enter the **Galaxy Server URL**, **Authentication Server URL**, and **API Token** created in the Prerequisite.
+.. Click btn:[Save].
+. Select the credential you created from the Organization’s Galaxy Credentials field.
+.. Navigate to menu:Access[Organizations].
+.. Select the organization where you’d like to add your Galaxy credentials.
+.. Click btn:[Edit].
+.. Under Galaxy Credentials, click image:search-icon.png.
```
Comment (sayjadha) [inline downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc:30]: I think for icon UI element, we state the icon name only. Consider revising '...click image:search-icon.png' -> ...click the **Search** icon.

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+[galaxy_server.<server_name>]
+-----
+
+. Set the `url` option for each server name. You must include the `api/galaxy/` subdirectory in the server URL:
++
+[subs="+quotes"]
+-----
+https://__<server_fully_qualified_domain_name>__/api/galaxy/
+-----
+. Set the `auth_url` option if necessary. The community Ansible Galaxy does not require an `auth_url`.
```
Comment (hherbly) [inline downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc:34]: Step 4 appears to be an optional step. I believe we have a specific formatting convention for optional steps (they should look something like "4. Optional: set the auth_url option..." See: https://docs.engineering.redhat.com/display/CCSENABLE/Optional+and+conditional+steps

**Review by hherbly — APPROVED** [review]

File: downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+== Configuring Red Hat automation hub as the primary source for content using the CLI
+
+Configure Red Hat automation hub as your primary source of content by using the command-line interface (CLI). To configure automation hub, you must modify the `ansible.cfg` configuration file. With automation hub, you have access to certified, supported collections.
+
+.Prerequisites
+
+* You have obtained the API token for the {HubName} server. See link: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/getting_started_with_red_hat_ansible_automation_hub/proc-create-api-token[Creating the {HubName} API token] for more information.
+[IMPORTANT]
+====
+	Creating a new token revokes any previous tokens
```
[the remark this answers is not in the document]
Comment (emurtoug) [reply downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc:15]: Hey Sayee! Thanks for this feedback. I'm keen to keep the wording of this as it is for clarity.

**Pull request merged**